### PR TITLE
Fixes chameleon projector immortality

### LINKED ISSUE #15065: Simple animals can't interact with chamelon projectors
Meaning they are complete invulnerability, even to player simple mobs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Allows simple animals, aliens, etc, to deactivate it by attacking it

Fixes #15065

### CODE REVIEW

**Comment by kevinz000** [comment]
I smell a salty demon that got sniped

**Comment by danhibiki** [comment]
this stops you from spamming spawnable mobs while under chameleon projection, right? :-1:

**Comment by ChangelingRain** [comment]
No, because you're not a valid target for them.
I'm not sure they'll attack it if you activate it while being targeted, but I'd assume they would.

**Comment by KorPhaeron** [comment]
This only applies to player simple animals, AI doesnt know to look for these holders

**Comment by YakumoChen** [comment]
:-1: zombies are too common from gold cores for gold-core bombing with a projector to be viable like this. It'll create player zombies, who will just salt and kill the projector user.

File: code/game/objects/items/devices/chameleonproj.dm [hunk 1]
```diff
@@ -64,6 +64,8 @@
 
 /obj/item/device/chameleon/proc/disrupt(delete_dummy = 1)
 	if(active_dummy)
+		for(var/mob/M in active_dummy)
+			M << "<span class='danger'>Your chameleon-projector deactivates.</span>"
```
Comment (tkdrg) [inline code/game/objects/items/devices/chameleonproj.dm]: this isn't a list
just do
`active_dummy << "<span class='danger'>Your chameleon-projector deactivates.</span>"`

**Comment by tkdrg** [comment]
Why can normal gold cores create player zombies without sentience potions, anyway? But that sounds like another issue unrelated to the fix at hand. This just forces you to treat player a simple_animal like a player human when it comes to hiding, which really isn't a problem as far as I'm concerned.

File: code/game/objects/items/devices/chameleonproj.dm (hunk 1 above)
Comment (duncathan) [inline code/game/objects/items/devices/chameleonproj.dm]: Lol you forgot about contents

Comment (tkdrg) [inline code/game/objects/items/devices/chameleonproj.dm]: I thought it was the mob and not the obj

**Pull request merged**